src/aws/lambda-complete/index.js

```diff
@@ -1,7 +1,6 @@
 'use strict';
```

Comment:
in commit message: 
s/we changed the video content-type/we were changing the video content-type